### PR TITLE
`WpfMath.Converters.SVGConverter` now enumerates resulting strings with `yield return` instead of adding to `StringBuilder`

### PR DESCRIPTION
**Pull request by Lehonti**
Also, I'm not sure why `m_nestedLevel` is needed, so I commented it out, but the code is still there if it's needed

### CODE REVIEW

File: src/WpfMath/Converters/SVGConverter.cs [hunk 1]
```diff
@@ -1,49 +1,50 @@
...
 using System.Globalization;
-using System.Text;
 using System.Windows.Media;
 
 namespace WpfMath.Converters;
 
 public class SVGConverter
 {
-    private int m_nestedLevel = 0;
+    //private int m_nestedLevel = 0;
```
Comment (ForNeVeR) [inline src/WpfMath/Converters/SVGConverter.cs:11]: Meh, it was never used, let's just drop it.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]
A bit of minor issues, everything else LGTM!

File: src/WpfMath/Converters/SVGConverter.cs (hunk 1 above)
Comment (Lehonti) [reply src/WpfMath/Converters/SVGConverter.cs]: Perfect, I just dropped it :) now let's just wait for the builds to finish :)

**Review by ForNeVeR — APPROVED** [review]
Great, thanks!

**Pull request merged**